services/catarse/spec/actions/billing/payments/authorize_transaction_spec.rb

```diff
@@ -68,7 +68,7 @@
         it 'updates payment gateway and gateway id' do
           result
 
-          expect(payment.reload.attributes).to include('gateway' => 'pagar_me', 'gateway_id' => gateway_response['id'])
+          expect(payment.reload.attributes).to include('gateway_id' => gateway_response['id'])
```

Comment:
**Sugestão**
Atualizar a descrição do `it` desse teste, pois ele não atualiza mais o gateway. A mesma sugestão se aplica para os testes do generate pix e o generate boleto.